src/ut/httpconnection_test.cpp

```diff
@@ -361,6 +361,17 @@ TEST_F(HttpConnectionTest, SimplePost)
   EXPECT_EQ(200, ret);
 }
 
+TEST_F(HttpConnectionTest, SimplePostWithHeaders)
+{
+  std::map<std::string, std::string> headers_in_rsp;
+  std::vector<std::string> headers_in_req;
```

Comment:
Worth including a header so that this is actually testing something?

Reply:
Good point, added.